### PR TITLE
Add Waku v2 plan

### PR DESCRIPTION
**Pull request by oskarth**
Add basic NOTE intro for post

Fix image linkes, markdown

### CODE REVIEW

File: _posts/2020-07-01-waku-v2-pitch.md [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+5.  [Next steps](#org1414f44)
+
+
+<a id="org0c3be09"></a>
+
+# Problem
+
+The Waku network is fragile and doesn't scale.
+
+As Status is moving into a user-acquisition phase and is improving retention rates for users we need the infrastructure to keep up, specifically when it comes to messaging.
```
Comment (decanus) [inline _posts/2020-07-01-waku-v2-pitch.md:52]:
```suggestion
As [Status](https://status.im) is moving into a user-acquisition phase and is improving retention rates for users they need the infrastructure to keep up, specifically when it comes to messaging.
```

File: _posts/2020-07-01-waku-v2-pitch.md [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+
+Based on user acquisition models, our initial goal is to support 100k DAU in September, with demand growing from there.
```
Comment (decanus) [inline _posts/2020-07-01-waku-v2-pitch.md:54]:
```suggestion
Based on user acquisition models, the initial goal is to support 100k DAU in September, with demand growing from there.
```

File: _posts/2020-07-01-waku-v2-pitch.md [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+# Appetite
+
+Our initial risk appetite for this is 6 weeks for a small team.
+
+The idea is that we want to make tangible progress towards the goal in a limited period of time, as opposed to getting bogged down in trying to find a theoretically perfect generalized solution. Fixed time, variable scope.
+
+It is likely some elements of a complete solution will be done separately. See later sections for that.
+
+
+<a id="org255ac65"></a>
```
Comment (decanus) [inline _posts/2020-07-01-waku-v2-pitch.md:101]: what is this?

File: _posts/2020-07-01-waku-v2-pitch.md [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+
+There are two main parts of the solution. One is to reduce amplification factors, and the other is incentivization to get more user run full nodes with desktop, etc.
+
+What does a full node provide? It provides connectivity to the network, can act as a bandwidth "barrier" and be high or reasonably high availability. What this means right now is essentially topic interest and storing historical messages.
+
+The goal is here to improve the status quo, not get a perfect solution from the get go. All of this can be iterated on further, for stronger guarantees, as well as replaced by other new modules.
+
+Let's first look at the baseline, and then go into some of the tracks and their phases. Track 1 is best done first, after which track 2 and 3 can be executed in parallel. Track 1 gives us more options for track 2 and 3. The work in track 1 is currently more well-defined, so it is likely the specifics of track 2 and 3 will get refined at a later stage.
+
+<a id="org9f0408b"></a>
```
Comment (decanus) [inline _posts/2020-07-01-waku-v2-pitch.md:113]: ditto

File: _posts/2020-07-01-waku-v2-pitch.md [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+3.  We assume the Stimbus production team will enable integration of nim-waku.
+4.  We assume Discovery mechanisms such as DNS and Discovery v5 will be worked on separately.
+5.  We assume Core will, at some point, provide an UX for integrating payment of services.
+6.  We assume the desktop client is sufficiently usable.
+7.  We assume Core and Infra will investigate ways of improving MaxPeers.
+
+
+<a id="org1414f44"></a>
+
+# Next steps
```
Comment (decanus) [inline _posts/2020-07-01-waku-v2-pitch.md:343]: Do we want this in the post

File: _posts/2020-07-01-waku-v2-pitch.md (hunk 3 above)
Comment (oskarth) [reply _posts/2020-07-01-waku-v2-pitch.md:101]: org mode links for clicks, removing

File: _posts/2020-07-01-waku-v2-pitch.md (hunk 5 above)
Comment (oskarth) [reply _posts/2020-07-01-waku-v2-pitch.md:343]: removed

File: _posts/2020-07-01-waku-v2-pitch.md [hunk 6]
```diff
@@ -0,0 +1,337 @@
...
+
+Figure 1-5:
+
+![](assets/img/status_scaling_model_fig1.png)
+![](assets/img/status_scaling_model_fig2.png)
+![](assets/img/status_scaling_model_fig3.png)
+![](assets/img/status_scaling_model_fig4.png)
+![](assets/img/status_scaling_model_fig5.png)
+
+See <https://colab.research.google.com/drive/1Fz-oxRxxAFPpM1Cowpnb0nT52V1-yeRu#scrollTo=Yc3417FUJJ_0> for the full report.
```
Comment (decanus) [inline _posts/2020-07-01-waku-v2-pitch.md:72]: do we want to share this link?

File: _posts/2020-07-01-waku-v2-pitch.md [hunk 7]
```diff
@@ -0,0 +1,337 @@
...
+    4.  [Track 3 - Accounting and user-run nodes](#org3b38a9b)
+        1.  [1. Adaptive nodes and capabilities](#org92e8676)
+        2.  [2. Accounting](#org376e97f)
+        3.  [3. Relax high availability requirement](#org18fd27b)
+        4.  [4. Incentivize light and full nodes to tell the truth (policy, etc)](#orge9bb567)
+        5.  [5. Settlement PoC](#org5b0cf19)
+4.  [Out of scope](#orgc50fab1)
+
+
+<a id="org0c3be09"></a>
```
Comment (decanus) [inline _posts/2020-07-01-waku-v2-pitch.md:45]: ditto.

File: _posts/2020-07-01-waku-v2-pitch.md (hunk 6 above)
Comment (oskarth) [reply _posts/2020-07-01-waku-v2-pitch.md:44]: why not?

File: _posts/2020-07-01-waku-v2-pitch.md (hunk 7 above)
Comment (oskarth) [reply _posts/2020-07-01-waku-v2-pitch.md]: removed

**Pull request merged**